src/test/java/uk/gov/hmcts/reform/sandl/snlrules/rules/listings/HearingTypeDoesNotMatchSessionCaseTypeTests.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.reform.sandl.snlrules.rules.listings;
+
+import org.drools.core.base.RuleNameEqualsAgendaFilter;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.api.runtime.KieSession;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.sandl.snlrules.model.HearingPart;
+import uk.gov.hmcts.reform.sandl.snlrules.model.Session;
+import uk.gov.hmcts.reform.sandl.snlrules.services.DroolsService;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+import static uk.gov.hmcts.reform.sandl.snlrules.rules.RulesTestHelper.assertProblems;
+
+@RunWith(MockitoJUnitRunner.class)
+public class HearingTypeDoesNotMatchSessionCaseTypeTests {
+
+    private final static String rulesDefinition = "Listings";
+
+    private DroolsService droolsService;
+    private KieSession rules;
+
+    @Before
+    public void setup() {
+        droolsService = new DroolsService(rulesDefinition);
+        droolsService.init();
+
+        rules = droolsService.getRulesSession();
+    }
+
+    @Test
+    public void should_be_no_problem_when_hearing_caseType_matches_session_caseType() {
+        String sessionId = "f9c351ec-0668-4000-9c39-927518f7389a";
```

Comment:
Ids are duplicated in the test below.

Reply:
I will thin whether it may be done slightly better but I would strongly recommend the following reading that explains DRY vs DAMP in tests 
https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests

Reply:
I would think that it's not the good idea to have an object with some common values and then change other values in every tests, but these are just strings. 
It's not a big deal, it can stay as it is for me :)

Reply:
some of that done